env.example

```diff
@@ -4,7 +4,8 @@ CSRF_TRUSTED_ORIGINS=
 DJANGO_SECRET_KEY=
 DJANGO_DEBUG=True
 
-POSTGRES_SERVER=
+POSTGRES_HOST=
 POSTGRES_DB=
 POSTGRES_USER=
 POSTGRES_PASSWORD=
+POSTGRES_DATA=
```

Comment:
Not sure how to handle the situation where the user wants to use a docker volume for db storage, because then there'd need to be a volume declaration at the bottom of the compose file. I think this is OK for now--in prod I'll use a docker volume and tweak the compose file accordingly.